.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
+
+[*]
+end_of_line = crlf
```

Comment:
I'm not sure if this is the best choice. Generally Git should be handling the line-endings. You even set `text=auto` in the gitattributes, which does the line ending normalization.

Reply:
Yeah, true. Thank you. 😄